document/CoreFormat.mdk

```diff
@@ -996,7 +1077,10 @@ This allows tools to include information about the result that is not explicitly
 
 A `location` object specifies the location where a static analysis tool produced a result.
 A `location` object should specify the physical location (&sect;[#physicalLocation]) of the result,
-and it may also specify the logical location (&sect;[#location-logicalLocation]) of the result.
+and it may also specify the logical location (&sect;[#location-fullyQualifiedLogicalName]) of the result.
+
+A logical location specifies a programmatic construct, for example, a class name or a function name,
```

Comment:
It's been there all along; I just moved it up here from further down. I think it says something useful ("don't include a file name in this thing!"). I'm open to suggestions for rephrasing it.